design-toolkit/components/package.json

```diff
@@ -114,7 +113,10 @@
     "@types/react": "^19.0.10",
     "@types/react-dom": "^19.0.4",
     "autoprefixer": "10.4.20",
+    "babel-plugin-react-compiler": "19.1.0-rc.2",
+    "esbuild": "0.25.6",
```

Comment:
TODO: ensure TSUP peer deps don't impact this

Reply:
We've been talking about trying out https://github.com/colinhacks/zshy as an alternative to `tsup`

Reply:
Nice yeah zshy does look incredibly simple (in a good way). It would likely be possible to run the compiler still even though zshy lacks plugin support by the looks of it. Basically would involve us running babel against the output from zshy. The rust variant of the React Compiler cannot come soon enough 😆